src/main/java/org/gridsuite/modification/server/modifications/OperatingStatusModification.java

```diff
@@ -151,9 +154,10 @@ private void applyEnergiseEquipmentEnd(Reporter subReporter, Identifiable<?> equ
 
         subReporter.report(Report.builder()
                 .withKey("energise" + equipmentType + "EndApplied")
-                .withDefaultMessage(equipmentType + " ${id} (id) : energise the side ${side} applied")
+                .withDefaultMessage("${equipmentType} ${id} (id) : energise the side ${side} applied")
                 .withValue("id", equipment.getId())
                 .withValue("side", side.name())
+                .withValue("equipmentType", equipmentType)
```

Comment:
would be better if the withValue statements follow the same left->right order

Reply:
Fixed